### DISCUSSION TITLE
关于SQL

### DISCUSSION THREAD

**Discussion by WT1W** [liusy58/NJU_AdvancedProgramming_SP22#74] [Q&A] [answered]
请问在SQL生成时，遇到一些问题还需要生成SQL吗，比如说下架商品，但不存在该商品，还需要生成SQL指令吗

**Comment by RicoloveFeng** [comment, marked as answer]
不

**Reply by WT1W** [reply]
那请问查看商品信息时找不到商品，也不用生成SQL指令吗？或者说这类找不到用户或商品的情况都不用生成SQL？

**Reply by binbin00** [reply]
只要最后没有对表进行任何操作，就不用生成指令。项目中对表的操作包括 插入、修改、打印表中信息。
